src/background/whotracksme.js

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
how about to call this file `tab-stats`?

Reply:
We of course can change it `tab-stats`, but then I would recommend to change the content script as well.

Reply:
please do